yoti/YotiWidget.php

```diff
@@ -26,8 +26,14 @@ public function widget($args, $instance)
     {
         wp_enqueue_style('yoti-asset-css', plugin_dir_url(__FILE__) . 'assets/styles.css');
         $config = YotiHelper::getConfig();
+        $widgetTitleHtml = '';
+        $widgetDefaultTitle = 'New title';
```

Comment:
Why not leave is as blank and add a title when the widget is being displayed, similar to other how other widgets are doing it. E.g. you add a title to the widget, the title gets displayed; you don't add a title, the field stays blank (without `New title`) and the widget displays a default title(`Authenticate with Yoti` maybe?)

Reply:
The default title is added by wordPress by default. I can set the default to `Authenticate with Yoti`. Also the default title `New title` won't get displayed if not title is set.